spec/support/fetch_table.rb

```diff
@@ -4,11 +4,7 @@ module FetchTable
   def fetch_table(selector)
     find(selector).all('tr').map do |row|
       row.all('th,td').map do |cell|
-        if cell.all('option').present?
-          cell.all('option').collect(&:text).join(' ')
-        else
-          cell.text.squish
-        end
+        cell.text.squish
       end
```

Comment:
```cell.all('option')``` would find all ```<option>``` elements within a cell, and then would go and extract the text from them, concatenating them together. However, if a cell didn't contain any option elements, it would wait until one appeared, or capybara timed out, before deciding that there weren't any. This meant that fetch table would end up waiting the whole capybara timeout per cell.

Meanwhile, cell.text already returns the concatenated values of any option tags within the cell. I can only assume that an older version of capybara or perhaps poltergeist behaved differently. Still, now the above step is entirely redundant.

Feature test:
Before: 223.03s
After: 6.87s